options.go

```diff
@@ -53,11 +56,12 @@ type Option func(*Options) error
 
 func GetDefaultOptions() Options {
 	return Options{
-		SendTimeout: 30 * time.Second,
-		IdleTime:    5 * time.Second,
-		ReadTimeout: 60 * time.Second,
-		PingHandler: nil,
-		TLSConfig:   nil,
+		ConnectTimeout: 10 * time.Second,
```

Comment:
This seems dangerous unless you are doing a major version bump

Reply:
In the context of this PR, the idea is that Pool will try to create a new connection after 10 seconds. I also don't think that it should take more than 10 seconds to connect to a brand network.

Reply:
Discussed, not a big deal, this would only impact a client that was using this library, and relying on the connection attempt not ever timing out or using the one from the OS